2019-03-04 08:17:40 [scrapy.utils.log] INFO: Scrapy 1.5.1 started (bot: ARGUS)
2019-03-04 08:17:40 [scrapy.utils.log] INFO: Versions: lxml 4.2.4.0, libxml2 2.9.5, cssselect 1.0.3, parsel 1.5.0, w3lib 1.19.0, Twisted 18.7.0, Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)], pyOpenSSL 18.0.0 (OpenSSL 1.1.0i  14 Aug 2018), cryptography 2.3.1, Platform Windows-7-6.1.7601-SP1
2019-03-04 08:17:40 [scrapy.crawler] INFO: Overridden settings: {'AJAXCRAWL_ENABLED': True, 'BOT_NAME': 'ARGUS', 'COMPRESSION_ENABLED': False, 'CONCURRENT_ITEMS': 200, 'CONCURRENT_REQUESTS': 100, 'COOKIES_ENABLED': False, 'DOWNLOAD_MAXSIZE': 10000000, 'DOWNLOAD_TIMEOUT': 20, 'FEED_URI': 'file:///C:/ARGUS-webarchive/ARGUS-webarchive/items/ARGUS/webarchive/967e3da63e4d11e9b5f800505680340c.jl', 'LOG_FILE': 'logs\\ARGUS\\webarchive\\967e3da63e4d11e9b5f800505680340c.log', 'LOG_LEVEL': 'INFO', 'NEWSPIDER_MODULE': 'ARGUS.spiders', 'REACTOR_THREADPOOL_MAXSIZE': 30, 'REDIRECT_MAX_TIMES': 2, 'RETRY_ENABLED': False, 'SPIDER_MODULES': ['ARGUS.spiders']}
2019-03-04 08:17:40 [py.warnings] WARNING: c:\python36\lib\site-packages\scrapy\utils\deprecate.py:156: ScrapyDeprecationWarning: `scrapy.contrib.feedexport.FeedExporter` class is deprecated, use `scrapy.extensions.feedexport.FeedExporter` instead
  ScrapyDeprecationWarning)

2019-03-04 08:17:40 [scrapy.middleware] INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2019-03-04 08:17:40 [scrapy.middleware] INFO: Enabled downloader middlewares:
['scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'scrapy_fake_useragent.middleware.RandomUserAgentMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.ajaxcrawl.AjaxCrawlMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2019-03-04 08:17:40 [scrapy.middleware] INFO: Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2019-03-04 08:17:40 [scrapy.middleware] INFO: Enabled item pipelines:
['ARGUS.pipelines.TextPipeline']
2019-03-04 08:17:40 [scrapy.core.engine] INFO: Spider opened
2019-03-04 08:17:40 [scrapy.extensions.logstats] INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-03-04 08:18:15 [scrapy.core.scraper] ERROR: Spider error processing <GET https://web.archive.org/web/20181012144529id_/https://www.zew.de/en/presse/> (referer: https://www.zew.de/en/presse/)
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 30, in process_spider_output
    for x in result:
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\ARGUS-webarchive\ARGUS-webarchive\ARGUS\spiders\webarchive.py", line 316, in processURLstack
    elif len(urlstack) == 0 and len(current_datestack) == 0:
NameError: name 'current_datestack' is not defined
2019-03-04 08:18:18 [scrapy.core.scraper] ERROR: Spider error processing <GET https://web.archive.org/web/20160130071316id_/http://www.zew.de/en/kontakt/> (referer: https://www.zew.de/en/kontakt/)
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 30, in process_spider_output
    for x in result:
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\ARGUS-webarchive\ARGUS-webarchive\ARGUS\spiders\webarchive.py", line 316, in processURLstack
    elif len(urlstack) == 0 and len(current_datestack) == 0:
NameError: name 'current_datestack' is not defined
2019-03-04 08:18:18 [scrapy.core.scraper] ERROR: Spider error processing <GET https://web.archive.org/web/20110417160745id_/http://www.zew.de/en/presse> (referer: https://www.zew.de/en/presse/)
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 30, in process_spider_output
    for x in result:
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\ARGUS-webarchive\ARGUS-webarchive\ARGUS\spiders\webarchive.py", line 316, in processURLstack
    elif len(urlstack) == 0 and len(current_datestack) == 0:
NameError: name 'current_datestack' is not defined
2019-03-04 08:18:18 [scrapy.core.scraper] ERROR: Spider error processing <GET https://web.archive.org/web/20160130071316id_/http://www.zew.de/en/kontakt/> (referer: https://www.zew.de/en/kontakt/)
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 30, in process_spider_output
    for x in result:
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\ARGUS-webarchive\ARGUS-webarchive\ARGUS\spiders\webarchive.py", line 316, in processURLstack
    elif len(urlstack) == 0 and len(current_datestack) == 0:
NameError: name 'current_datestack' is not defined
2019-03-04 08:18:19 [scrapy.core.scraper] ERROR: Spider error processing <GET https://web.archive.org/web/20181019092200id_/https://www.zew.de/en/kontakt/> (referer: https://www.zew.de/en/kontakt/)
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 30, in process_spider_output
    for x in result:
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\ARGUS-webarchive\ARGUS-webarchive\ARGUS\spiders\webarchive.py", line 316, in processURLstack
    elif len(urlstack) == 0 and len(current_datestack) == 0:
NameError: name 'current_datestack' is not defined
2019-03-04 08:18:19 [scrapy.core.scraper] ERROR: Spider error processing <GET https://web.archive.org/web/20160331050829id_/http://www.zew.de/en/kontakt/> (referer: https://www.zew.de/en/kontakt/)
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 30, in process_spider_output
    for x in result:
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\python36\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\ARGUS-webarchive\ARGUS-webarchive\ARGUS\spiders\webarchive.py", line 316, in processURLstack
    elif len(urlstack) == 0 and len(current_datestack) == 0:
NameError: name 'current_datestack' is not defined
2019-03-04 08:18:19 [scrapy.core.engine] INFO: Closing spider (finished)
2019-03-04 08:18:19 [scrapy.statscollectors] INFO: Dumping Scrapy stats:
{'downloader/request_bytes': 12147,
 'downloader/request_count': 37,
 'downloader/request_method_count/GET': 37,
 'downloader/response_bytes': 3192644,
 'downloader/response_count': 37,
 'downloader/response_status_count/200': 32,
 'downloader/response_status_count/301': 4,
 'downloader/response_status_count/403': 1,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2019, 3, 4, 7, 18, 19, 648307),
 'log_count/ERROR': 6,
 'log_count/INFO': 7,
 'log_count/WARNING': 1,
 'request_depth_max': 8,
 'response_received_count': 35,
 'scheduler/dequeued': 37,
 'scheduler/dequeued/memory': 37,
 'scheduler/enqueued': 37,
 'scheduler/enqueued/memory': 37,
 'spider_exceptions/NameError': 6,
 'start_time': datetime.datetime(2019, 3, 4, 7, 17, 40, 717414)}
2019-03-04 08:18:19 [scrapy.core.engine] INFO: Spider closed (finished)
